src/wechaty.ts

```diff
@@ -683,13 +683,13 @@ export class Wechaty extends Accessory implements Sayable {
             const inviter = this.Contact.load(inviterId)
             await inviter.ready()
 
-            this.emit('room-join', room, inviteeList, inviter)
-            room.emit('join', inviteeList, inviter)
+            this.emit('room-join', room, new Date(timestamp), inviteeList, inviter)
```

Comment:
save `new Date(timestamp)` to a variable and reuse it instead of call `new Date` twice.

Reply:
I would prefer to have different variables, since users might modify the value of the variable(you never know).
If they listened on `wechat`'s `room-join` events, also listened to the specific `room`'s `room-join` event, modifying one time will have effect on the other one, which might cause some very unexpected bug in the future.

Alternative solution: we could make this value readonly or freeze this time variable.

What do you think about this?

Reply:
I don't think this will happen because the user can only modify their local copy of the variable value.